conf/dnsmasq/domain.tpl

```diff
@@ -5,5 +5,9 @@ interface-name={{ domain }},{{ interface }}
 {% if ipv6 %}
 host-record={{ domain }},{{ ipv6 }}
 {% endif %}
+{% if mail_out == "1" %}
 txt-record={{ domain }},"v=spf1 mx a -all"
```

Comment:
What if another server is used to send mails to for a domain ?
For example we have a serverA, a yunohost with mailbox toto@domain.tld

And a serverB, with a website that send emails noreply@domain.tld to toto@domain.tld.

The official zone includes the IP of the mailserver of serverB in SPF, but here we have our dnsmasq lying, so i guess the postfix of ynh serverA could refuse the mail !

Reply:
In fact, i think this line is only here for the usecase where DNSmasq is configured as the authority DNS server...

Reply:
I think i will creates an other PR for this...